Tasks/Common/azure-arm-rest/webClient.ts

```diff
@@ -0,0 +1,86 @@
+import tl = require('vsts-task-lib/task');
+import util = require("util")
+var httpClient = require('vso-node-api/HttpClient');
```

Comment:
Not directly related to this PR, but why do we need vso-node-api dependency here? All we need is some http client. we can directly use what vso-node-api is using - i think it is `'typed-rest-client/HttpClient'`